pages/common/lsof.md

```diff
@@ -8,8 +9,16 @@
 
 - find the process that opened a local internet port
 
-`lsof -i :{{8080}}`
+`lsof -i :{{port}}`
 
 - only output the process PID (e.g. to pipe into kill)
 
-`lsof -t {{/path/to/file}} | xargs kill -9`
+`lsof -t {{/path/to/file}}`
```

Comment:
While you forgot to remove the comment in line 14, `(e.g. to pipe into kill)`, it looks quite natural. I'd say it can stay but I'll let some of the other @maintainers decide.